tests/cupyx_tests/scipy_tests/linalg_tests/test_matfuncs.py

```diff
@@ -0,0 +1,32 @@
+from matfuncs import sinhm
+import numpy
+import unittest
+
+
+class TestMatFuncsNan(unittest.TestCase):
```

Comment:
It seems your test cases only cover anomaly cases, and using NumPy array as an input.

Reply:
I was converting array in input as NumPy array to use functions defined for NumPy array like shape.

Reply:
Which other test cases should i also add?

Reply:
I have edited the files .
Please guide me for any further changes .

Reply:
@ameyachawla99 Did you check if this test passes locally?

Reply:
Yes all 5 tests were passes 
i have added the screenshot of terminal .
![Screenshot from 2021-03-27 12-25-38](https://user-images.githubusercontent.com/78856161/112712827-d49c2000-8ef7-11eb-845b-11c14c1932ed.png)

Reply:
Why `from matfuncs import sinhm` works? `matfuncs.py` is not placed in test directory.

Reply:
Anyway, this test code looks to just call NumPy routines. Could you use `cupy.ndarray` for the input?

Reply:
> Why `from matfuncs import sinhm` works? `matfuncs.py` is not placed in test directory.

While i was testing i placed matfuncs in same directory .

Reply:
> call NumPy routines

Sure i will update it .

Reply:
> Anyway, this test code looks to just call NumPy routines. Could you use `cupy.ndarray` for the input?

But then changes has to be made in both files as error will be caused of cupy.ndarray as not of same datatype of numpy.ndarray

Reply:
@asi1024 i have made the changes in both files so that they can use cupy.ndarray instead of numpy.ndarray

Reply:
Could you add an alias `cupyx.scipy.linalg.sinhm` and use it in the test?

Reply:
SciPy's documentation says that `scipy.linalg.sinhm` uses **matrix exponentials**, but your implementation uses element-wise exponential.  https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.sinhm.htm.

Reply:
Then i will first define expm which is used 

> SciPy's documentation says that `scipy.linalg.sinhm` uses **matrix exponentials**, but your implementation uses element-wise exponential. https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.sinhm.htm.

Then i will first define expm

Reply:
Oh, sorry I overlooked that `cupyx.scipy.sparse.linalg.expm` is not implemented yet... I should have noticed it and comment that in the first review round. You can either try implementing it in this PR or in a separate PR.

Reply:
Actually it is my fault i did not understood the function properly